src/api/decisionsApi.tsx

```diff
@@ -62,7 +60,7 @@ export type DecisionPost  ={
   };
   export const statusTypeGetParser = (statusType: number): string =>{
     if(statusType === 0) return "У розгляді";
-    if (statusType === 1) return "Підтверджено";
+    if (statusType === 1) return "Потверджено";
```

Comment:
just out of curious, is this really how Plast call something is approved?😂😂😂

Reply:
Yes, this is Old Slavic or something like that))

Reply:
Got you
For we it's really weird

Reply:
Maybe you want also review and approve this pull request?)

Reply:
Sure